### DISCUSSION TITLE
Users in pretix

### DISCUSSION THREAD

**Discussion by Duncan-Kiragu** [pretix/pretix#3157] [Q&A]
Hello,

Is there a way to make users global rather than tied to a specific organizer?

**Comment by pc-coholic** [comment]
Users are always global to the whole pretix instance.

Any user can also be a member of one or more teams which in turn are tied to an organizer.

By default, when inviting a user, they get automatically added to a team and thus organizer. But there is nothing stopping you from adding that user to more teams and by extension more organizers.

**Reply by Duncan-Kiragu** [reply]
I think I phrased my question wrongly. 

My concern is the customers who are buying tickets - they are tied to a specific organizer rather than to the platform. I am looking for a way to make the customers global in that when they sign up - they can buy different tickets from different event organizers under their account.

**Reply by raphaelm** [reply]
Sorry, there's no way to do that

**Reply by Duncan-Kiragu** [reply]
Alright
